lib/scrabble.rb

```diff
@@ -9,5 +12,15 @@ def point_values
       "U"=>1, "V"=>4, "W"=>4, "X"=>8,
       "Y"=>4, "Z"=>10
     }
+    @scored_letters = scored_letters
+  end
+
+  def score(word)
+    letters_to_score = word.upcase.chars
+    @scored_letters = letters_to_score.map do |letter|
+     @point_values[letter]
+    end
+    sum = 0
+    total_value = @scored_letters.map {|score| sum+=score}
```

Comment:
this is soooooo close!! all you need to do is call the `sum` variable at the end of the method to make that sum the return value of the method